Library/Homebrew/tab.rb

```diff
@@ -240,9 +240,11 @@ def without?(val)
     !with?(val)
   end
 
+  # rubocop:disable Rails/Delegate
```

Comment:
Out of interest: why disable here?

Reply:
I commented it out temporarily.
I'll remove these comments and change `delegator`.